packages/ai-core/README.md

```diff
@@ -0,0 +1,166 @@
+              "key": "modelName",
+              "value": "gpt-35-turbo"
+            },
+            {
+              "key": "modelVersion",
+              "value": "latest"
+            }
+        ],
+        inputArtifactBindings: []
+    }
+
+    try {
+        const responseData: ConfigurationCreationResponse = await ConfigurationApi
+            .configurationCreate(requestBody, {'AI-Resource-Group': 'default'})
+            .execute();
+        return responseData;
+    } catch (errorData) {
+        const apiError = errorData.response.data.error as ApiError;
+        console.error('Status code:', errorData.response.status);
+        throw new Error(`Configuration creation failed: ${apiError.message}`);
+    }
+}
+```
+
+### DeploymentApi
+
+#### Create a Deployment
+
+```TypeScript
+async function createDeployment() {
+
+    const requestBody: DeploymentCreationRequest = {
+      configurationId: '0a1b2c3d-4e5f6g7h'
+    };
+
+    try{
+        const responseData: DeploymentCreationResponse = await DeploymentApi
+            .deploymentCreate(requestBody, {'AI-Resource-Group': 'default'})
+            .execute();
+        return responseData;
+    } catch (errorData) {
+        const apiError = errorData.response.data.error as ApiError;
+        console.error('Status code:', errorData.response.status);
+        throw new Error(`Deployment creation failed: ${apiError.message}`);
+    }
+}
+```
+
+#### Delete a Deployment
+
+Only deployments with `targetStatus: STOPPED` can be deleted. So a modification request must be sent before deletion can occur.
+
+```TypeScript
+async function modifyDeployment() {
+
+    let deploymentId: string = '0a1b2c3d4e5f';
+
+    const deployment: DeploymentResponseWithDetails = await DeploymentApi
+        .deploymentGet(deploymentId, {}, {'AI-Resource-Group': 'default'})
+        .execute();
+
+    if(deployment.targetStatus === 'RUNNING') {
+        // Only RUNNING deployments can be STOPPED.
+        const requestBody: DeploymentModificationRequest = {
+            targetStatus: 'STOPPED',
+        };
+
+        try {
+            await DeploymentApi
+                .deploymentModify(deploymentId, requestBody, {'AI-Resource-Group': 'default'})
+                .execute();
+        } catch (errorData) {
+            const apiError = errorData.response.data.error as ApiError;
+            console.error('Status code:', errorData.response.status);
+            throw new Error(`Deployment modification failed: ${apiError.message}`);
+        }
+    }
+    // Wait a few seconds for the deployment to stop
+    try {
+        return DeploymentApi.deploymentDelete(deploymentId, { 'AI-Resource-Group': 'default' }).execute();
+    } catch (errorData) {
+        const apiError = errorData.response.data.error as ApiError;
+        console.error('Status code:', errorData.response.status);
+        throw new Error(`Deployment deletion failed: ${apiError.message}`);
+    }
+}
+```
+
+## Support, Feedback, Contribution
```

Comment:
[q] Is this required to be present in all README's ? Can we just restrict it to the root README?